### PR TITLE
Plans: update grid for introductory offers

### PR DESCRIPTION
**Pull request by michaeldcain**
This updates the plans grid to properly account for different introductory offers for WPcom plans. It updates the Introductory Offer plan descriptions and consolidates some of the display logic between discounts and introductory offers.

> [!NOTE]
> The actual text for the offers is a bit long, includes awkward newlines, and could be improved upon. However, that is also true of the existing text returned by `usePlanBillingDescription()` and this is primarily a bug fix for missing behavior (this will allow us to remove an old "temporary" discount and replace it with introductory offers) and is not designed to make the text follow any new designs. Follow-up PRs can be made after this merges.

Requires: D134559-code
See: pbOQVh-4sd-p2

Before             |  After
:-------------------------:|:-------------------------:
Screenshot 2024-10-04 at 3 17 54 PM | Screenshot 2024-10-07 at 4 44 08 PM
Screenshot 2024-10-03 at 5 16 59 PM | Screenshot 2024-10-07 at 4 44 19 PM

Here's the logged-out view:

Before             |  After
:-------------------------:|:-------------------------:
logged-out-before | logged-out-after

**To test:**
- enable Store Sandbox and make sure that `public-api.wordpress.com` is pointing to your sandbox.
- make sure D134559-code is applied to your sandbox.
- create a new user, or use a user that has not previously subscribed to one of these plans
- change your user currency to MXN, PHP, or INR
- create a new site and verify that the different plans grids correctly reflect the introductory offer pricing (reduced monthly breakdown, intro offer full price in the description, and correct billing term in the description)

### CODE REVIEW

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [inline client/my-sites/plans-grid/components/header-price.tsx]: Not sure I follow why we are removing this. Does it no longer break the layout? (uou can follow the PR that added this to test, also same for badges).

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [inline client/my-sites/plans-grid/components/header-price.tsx]: I think this was 0 on purpose, as it's never visible. It's a placeholder price to fix some design issues. Do you prefer setting an actual price to it?

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [inline client/my-sites/plans-grid/components/header-price.tsx]: The badge was updated initially to match the WooExpress $1 intro offers, which had clear mocks that were communicated/followed (hence the `isForIntroOffer` parts). These changes will break that design too. Are we decided/aligned with Woo on this?

You can test the WooExpress $1 intro offers through `/setup/wooexpress` then `/plans` with that site.

**Review by chriskmnds — CHANGES_REQUESTED** [review]
Thanks for working on this. I left some comments in the code that may need addressing before merging.

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [reply client/my-sites/plans-grid/components/header-price.tsx]: Ok, I missed reading the part about "crossout" pricing. I think we still need to redo as we render a single price below in the else clause. Why do we leave that, and do we only render crossout for the minimized prices (when `isLargeCurrency` is true), as that seems to be the case now.

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (michaeldcain) [reply client/my-sites/plans-grid/components/header-price.tsx]: This felt like an unnecessary fork in the design logic that will need to be remembered and kept up in the future. Do you know if there was some reasoning behind its introduction?

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (michaeldcain) [reply client/my-sites/plans-grid/components/header-price.tsx]: This felt like another unnecessary fork in the display logic. I opted to remove both of these because they make the logic more complex, the variable naming incorrect (Woo is not the only service that can use introductory offers), and make future updates to the plan grids take more time. At the end of the day, I'm fine with adding these elements back, but I think we should be careful about introducing forks in the display logic.

My reasoning was that if there's a cost reduction, it makes sense to show it, unless there's been a specific discussion on why we shouldn't for introductory offers. The billing functionality for these introductory offers and a new purchase discount is essentially the same.

I left the actual styling for `is-placeholder-price` in place, in case there's a need to render the page without the crossed-out price (as we do when `isLargeCurrency` is false).

**Comment by michaeldcain** [comment]
@chriskmnds thanks for the review! I tried to justify my changes in some comment, but I'd be happy to revert them if you'd like me to.

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [reply client/my-sites/plans-grid/components/header-price.tsx]: I will take a closer look tomorrow. The `is-placeholder-price` logic/styling was introduced to align prices across the grid when only some of the prices are discounted or shown crossed-out (the ones that weren't would flow in the middle otherwise). The same applies for badges.

In this case, what i noticed is you only show crossed-out price when `isLargeCurrency`, otherwise it will shkw a single price. `isLargeCurrency` is true when the whole string exceeds certain number of characters, so it shouldn't affect the logic.

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [reply client/my-sites/plans-grid/components/header-price.tsx]: Yes, there were design mocks that we followed. I recall some discussions around color, size, etc. I will dig out the P2 post and share tomorrow.

**Comment by chriskmnds** [comment]
@michaeldcain Sorry for delay in reply. If you can wait another day i can take a closer look and share the original reasoning/posts behind the existing styling. My impression is we at least need to notify WooExpress designers about changing that badge styling. We could also create a separate one if we need to diverge from the all-caps blue-letters badge.

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [reply client/my-sites/plans-grid/components/header-price.tsx]: > In this case, what i noticed is you only show crossed-out price when isLargeCurrency, otherwise it will shkw a single price. isLargeCurrency is true when the whole string exceeds certain number of characters, so it shouldn't affect the logic.

Confirmed and this is indeed the case right now. You only apply the cross-out pricing for `isLargeCurrencuy` (meaning, for price strings that exceed a certain number of characters, which get a lower font size), while the other case will show a single price unit. This is how the intro offers for a Woo site looks with these changes (no cross-out, as it never reaches the character threshold):

Screenshot 2024-01-26 at 12 07 01 PM

My suggestion is to either revert this change (so to not show crossed-out prices) or to remove the `isLargeCurrency` check and the single `PlanPrice` instance below (so to always show crossed-out pricing). Otherwise it can lead to inconsistencies - users seeing either/or depending on the price, currency, and any other discount applied (which may force decimals on the price unit).

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [reply client/my-sites/plans-grid/components/header-price.tsx]: This was the main PT: peapX7-2Dm-p2
The design mocks were communicated/discussed here: peapX7-352-p2#comment-4387

So maybe reach out to one of the designers, and/or @warrendholmes / @ilyasfoo to confirm they are ok with the updated designs.

**Comment by michaeldcain** [comment]
@chriskmnds thanks for all of the additional context. I'll make some additional changes this week and won't merge this without an approval from you.

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (ilyasfoo) [reply client/my-sites/plans-grid/components/header-price.tsx]: > So maybe reach out to one of the designers, and/or @warrendholmes / @ilyasfoo to confirm they are ok with the updated designs.

I think the new design looks good. I'd agree with better maintainability. 

cc @verofasulo if you have an opinion, I've attached the difference in the following screenshot comparison of the plans grid for Woo Express $1 offer:


| Before      | After |
| ----------- | ----------- |
| image | image   |

**Comment by michaeldcain** [comment]
@chriskmnds after seeing a number of new WPcom introductory offers being discussed (e.g. the new Migration Creator trial), I dusted this off and went ahead and rebased it and the backend diff. 

Given that Woo Express plans have been replaced with WPcom e-commerce trials/introductory offers, and there wasn't [any pushback to the design changes above](https://github.com/Automattic/wp-calypso/pull/86434#discussion_r1470822660), I think it might make sense to proceed with the changes here. Do you mind giving it a review?

**Comment by chriskmnds** [comment]
> @chriskmnds after seeing a number of new WPcom introductory offers being discussed (e.g. the new Migration Creator trial), I dusted this off and went ahead and rebased it and the backend diff.
> 
> Given that Woo Express plans have been replaced with WPcom e-commerce trials/introductory offers, and there wasn't [any pushback to the design changes above](https://github.com/Automattic/wp-calypso/pull/86434#discussion_r1470822660), I think it might make sense to proceed with the changes here. Do you mind giving it a review?

Nice. Yes, having a single base style/design for these makes sense. There was a part where I noticed the cross-out pricing only showed for large-currency, so I'm looking into verifying that. I'll post an update in a bit.

File: packages/plans-grid-next/src/components/header-price.tsx
Comment (chriskmnds) [inline packages/plans-grid-next/src/components/header-price.tsx]: With "only" the changes below, you will only show the cross-out pricing when `isLargeCurrency` is `true`. I think what you want here is to remove this condition and have it work identical to the `isGridPlanOneTimeDiscounted` below (since intro offers will now always show cross-out price)?

**Review by chriskmnds — COMMENTED** [review]
Thanks for the ping. I left a comment in the code, which I think it's the same situation that I commented about back when reviewing this initially. 

Curious if we should also remove the `isGridPlanOneTimeDiscounted` blocks/logic as we don't render any other form of "on-time discount" in the grid right now (this was the only case). I can create a follow-up issue to investigate once this is deployed. Let me know pls.

I also can't get the intro offers to show, but not sure if it's due to what I commented on or something with the endpoint not returning the intro offers. This is my view when sandboxing API and applying the diff:

Screenshot 2024-07-24 at 1 10 57 PM

Not sandboxing shows the discounts like before:

Screenshot 2024-07-24 at 1 27 50 PM

File: packages/plans-grid-next/src/components/header-price.tsx
Comment (chriskmnds) [reply packages/plans-grid-next/src/components/header-price.tsx]: I think it is same case as commented previously: https://github.com/Automattic/wp-calypso/pull/86434#discussion_r1467455621

**Comment by michaeldcain** [comment]
@chriskmnds Thanks for the review! Sorry I missed something from your initial feedback. I'll take a look shortly.

> I also can't get the intro offers to show

Is this for a new or existing user? Can you share the username? One of the biggest differences between using the hardcoded cost override and introductory offers is that introductory offers guard against a user cancelling a plan and then resubscribing with the same offer (something the hardcoded override doesn't account for). If your user has ever had a subscription for one of these plans in the past, the introductory offer won't be available.

**Comment by chriskmnds** [comment]
> Is this for a new or existing user? Can you share the username? 

Existing user - `chriskmnds`

> One of the biggest differences between using the hardcoded cost override and introductory offers is that introductory offers guard against a user cancelling a plan and then resubscribing with the same offer (something the hardcoded override doesn't account for). If your user has ever had a subscription for one of these plans in the past, the introductory offer won't be available.

This sounds like the reason for not seeing them. These additional restrictions that you mention, do they carry over to the fist-year currency discounts? We haven't had these same restrictions so far right?

p.s. something's not quite right on my end now - in production, I see the intro offers rendering like this (not through this PR):

Screenshot 2024-07-29 at 6 59 09 PM

Not sure if we shipped something in the tables that produce these intro offers, or if something's off with my account. Can you confirm if production (`/start/plans`) renders the currency discounts as previously?

-------------

> Curious if we should also remove the `isGridPlanOneTimeDiscounted` blocks/logic as we don't render any other form of "on-time discount" in the grid right now (this was the only case). I can create a follow-up issue to investigate once this is deployed. Let me know pls.

Never mind this comment. We render the "one time discount" label for coupon discounts, so there's still use for them.

**Comment by michaeldcain** [comment]
> These additional restrictions that you mention, do they carry over to the fist-year currency discounts? We haven't had these same restrictions so far right?

The introductory offer eligibility restrictions close a loophole in the logic that the previous first-year currency discounts had. The Woo Express discounts were always introductory offers, so behaved in the same way as the new currency-based introductory offers. 

The reasons for migrating the first-year currency discounts to introductory offers can be seen in many of the issues from this post: p58i-fkS-p2

> p.s. something's not quite right on my end now - in production, I see the first-year currency discounts rendering like intro offers, like this (not through this PR):

I think that you're seeing a mix of the currency-based introductory offers (by having Store Sandbox enabled) and the first-year cost overrides, but in production they render with the same logic that was used to show the Woo Express introductory offers. This kind of confusion is why I want to get this PR wrapped up.

Here are the two variations based on Store sandbox vs production tables (there's an indicator in the environment badge):

| Production Tables      | Store Sandbox Tables |
| ----------- | ----------- |
| Screenshot 2024-07-29 at 2 24 17 PM | Screenshot 2024-07-29 at 2 24 01 PM |

File: packages/plans-grid-next/src/components/header-price.tsx
Comment (jjchrisdiehl) [reply packages/plans-grid-next/src/components/header-price.tsx]: > I think what you want here is to remove this condition and have it work identical to the isGridPlanOneTimeDiscounted below (since intro offers will now always show cross-out price)?

I did a bit of testing here and this does work well and will enforce that both intro offers and discount prices use the same pricing format (i.e. small, crossed out price pairs).

We could also refactor `isGridPlanOnIntroOffer` and `isGridPlanOneTimeDiscounted` into one render block but maybe its cleaner just to have them separate alongside `isAnyVisibleGridPlanOneTimeDiscounted || isAnyVisibleGridPlanOnIntroOffer`.

**Review by jjchrisdiehl — COMMENTED** [review]
Functional testing, after following your steps I landed on the plans page and the green `LIMITED TIME OFFER` badges:

image

I think the code looks good and I agree with chriskmnds that removing the `isLargeCurrency` logic from the`isGridPlanOnIntroOffer` would be a benefit, since we always want this block to show the crossed-out pricing regardless of the price string length.

**Comment by jeyip** [comment]
Hi @michaeldcain!

I'm being asked to shape a second attempt at intro offers for annual plans ( p5uIfZ-fwN-p2#comment-23600 ). You can see the shaping doc here 1gTMlUooqddT2PJ5UNvcMdMlVfCaeRUqSrAA4Tpbx7vI-gdoc. We'll probably run the intro offers as an experiment, and the results will be used to help inform how we'd like to handle pricing in the larger [Dotcom Packaging 2024 project](https://docs.google.com/spreadsheets/d/1Q4PPP6Xb3IBk5mQ0kSe-a_b93LlTrmL6QZ-0v3Cbs2s/edit?gid=0#gid=0).  

If I understand this PR and this diff D134559-code correctly, we want to consolidate intro offer logic and one-time currency discount logic ( which are currently treated differently ). If this is the case do you happen to have a sense of how much more effort it would take to get these changes over the finish line?

**Comment by michaeldcain** [comment]
> If I understand this PR and this diff D134559-code correctly, we want to consolidate intro offer logic and one-time currency discount logic ( which are currently treated differently ). If this is the case do you happen to have a sense of how much more effort it would take to get these changes over the finish line?

@jeyip the backend diff is ready to go. I think this needs a rebase and then to address the feedback in [this comment](https://github.com/Automattic/wp-calypso/pull/86434#issuecomment-2256320395). I'd guess it could be ready in less than 2 days.

**Comment by jeyip** [comment]
Thanks for the update @michaeldcain!

File: packages/plans-grid-next/src/components/header-price.tsx
Comment (chriskmnds) [reply packages/plans-grid-next/src/components/header-price.tsx]: @jeyip I think fixing the large-currency inconsistency is the only part left to address (and a rebase obviously)

**Comment by sirbrillig** [comment]
I'm not certain what happened to this PR (I was on sabbatical at the time) but clearly the "2 days" it was going to take have expired 😅 . It's currently needed for a new project (offering discounted pricing for IDR and THB) so I thought I'd resurrect it and complete the work. I've rebased the changes and fixed merge conflicts on both this PR and D134559-code. 

I also updated the screenshots in the description to compare the current look.

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (sirbrillig) [reply client/my-sites/plans-grid/components/header-price.tsx]: From the code, it looks like this comment was never addressed. @chriskmnds if you remember this ancient comment... 😓 

> remove the isLargeCurrency check and the single PlanPrice instance below (so to always show crossed-out pricing).

Did you mean to remove `isLargeCurrency` for all the options on this page, or only the `isGridPlanOnIntroOffer` block? I think probably you meant only this block.

Comment (sirbrillig) [reply client/my-sites/plans-grid/components/header-price.tsx]: I've pushed a change to always display the crossed-out price for introductory offers that hopefully should resolve this issue.

File: packages/plans-grid-next/src/components/shared/header-price/index.tsx
Comment (chriskmnds) [inline packages/plans-grid-next/src/components/shared/header-price/index.tsx]: Should this be conditioned instead, like:

```
className={ clsx( 'plans-grid-next-header-price__pricing-group', {
    'is-large-currency': isLargeCurrency,
} ) }
```

as done below for the other block (`isGridPlanOneTimeDiscounted`)?

File: client/my-sites/plans-grid/components/header-price.tsx
Comment (chriskmnds) [reply client/my-sites/plans-grid/components/header-price.tsx]: Hey, @sirbrillig - Yes, this is what I meant. Thanks for addressing :-)

**Review by chriskmnds — COMMENTED** [review]
Thanks for the ping and for following up. Code looks good, I was also finally able to test. 

So all good code-wise. The only thing I notice is a slight design detail in the billing description. It looks a bit weird with the line wrap:

Screenshot 2024-10-04 at 3 29 27 PM

File: packages/plans-grid-next/src/components/shared/header-price/index.tsx
Comment (sirbrillig) [reply packages/plans-grid-next/src/components/shared/header-price/index.tsx]: Good catch!

**Comment by sirbrillig** [comment]
> So all good code-wise. The only thing I notice is a slight design detail in the billing description. It looks a bit weird with the line wrap

That's a good point! The issue is that all the strings in `usePlanBillingDescription()` have a `br` tag hard-coded inside them. IMO this is a poor idea since it puts a style issue in the hands of a (localized) string instead of at the level of CSS. I'd be inclined to remove all those `br` tags, but for two things:

1. The `br` tags exist before this PR as well and cover a bunch of strings this PR does not modify.
2. It still doesn't look ideal to me with the `br` tags removed and probably could use some additional styling changes to compensate.

So I'm not sure this is the PR to fix that. Maybe a follow-up? What do you think @chriskmnds?

For reference, here's how it would look with the `br` tags removed:

Screenshot 2024-10-04 at 11 28 59 AM

**Review by chriskmnds — APPROVED** [review]
@sirbrillig Thanks for the reply!

> That's a good point! The issue is that all the strings in `usePlanBillingDescription()` have a `br` tag hard-coded inside them. IMO this is a poor idea since it puts a style issue in the hands of a (localized) string instead of at the level of CSS. I'd be inclined to remove all those `br` tags, but for two things:
> 
> 1. The `br` tags exist before this PR as well and cover a bunch of strings this PR does not modify.
> 2. It still doesn't look ideal to me with the `br` tags removed and probably could use some additional styling changes to compensate.
> 

Yep. I think adding the price after `per month` in this PR exacerbates the issue more. Do we need that btw? Can't we keep the first line as `per month, for your first year`, like it was before?  

But I agree - it's poor design to embed `<br />` tags this way. Wrapping the parts and conditioning the styles might be better, like you mentioned, so definitely something to consider there. 👍 

> So I'm not sure this is the PR to fix that. Maybe a follow-up? What do you think @chriskmnds?

Follow-up sounds good! Let's merge this and the backend so we have things in place. I can summarize and create follow-up issue on Monday, unless you want to do it before. We can add it to MarTech/Decepticons board, and hopefully prioritize

**Comment by sirbrillig** [comment]
> Yep. I think adding the price after per month in this PR exacerbates the issue more. Do we need that btw? Can't we keep the first line as per month, for your first year, like it was before?

Yeah, I think that's a good idea. We can always update strings later. I'll make that change before going forward.

**Comment by sirbrillig** [comment]
I updated the screenshots in the description to show how it looks now with the price after the intro offer removed. I'm not sure it's the best idea but it keeps things looking the same. We can adjust it later.

**Comment by chriskmnds** [comment]
Hey @sirbrillig 

> I updated the screenshots in the description to show how it looks now with the price after the intro offer removed. I'm not sure it's the best idea but it keeps things looking the same. We can adjust it later.

Seen it. FWIW - without looking closer - I was referring to _just_ the price part next to the "per month", not the whole last line. 

So instead of:

```
per month, MX$432 for your first year,
then MX$540 billed annually, excl. taxes
```

to have:

```
per month, for your first year,
then MX$540 billed annually, excl. taxes
```

I think that's how it was before? 🤔

**Comment by sirbrillig** [comment]
> I think that's how it was before? 

I can do that if you want, but that's not how it is in production now. See the "Before" screenshot.

**Comment by sirbrillig** [comment]
Oh, you might see what you are describing if you don't have the backend patch applied when testing. I'll update the testing instructions to include mentioning that.

File: packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx
Comment (chriskmnds) [inline packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx]: Not sure I follow why this is necessary? Can we not leave it the same it was before?

```
return translate(
						'per month, for your first %(introOfferIntervalUnit)s,{{br/}}' +
							'then %(rawPrice)s billed annually, excl. taxes',
						{
							args: {
								rawPrice: originalPriceFullTermText,
								introOfferIntervalUnit: introOffer.intervalUnit,
							},
							components: { br: <br /> },
							comment: 'excl. taxes is short for excluding taxes',
						}
					);
```

File: packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx
Comment (chriskmnds) [inline packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx]: Same: https://github.com/Automattic/wp-calypso/pull/86434/files#r1789913879

Why change the original?

**Comment by chriskmnds** [comment]
@sirbrillig 

> Oh, you might see what you are describing if you don't have the backend patch applied when testing or if it's applied when you are testing production. I'll update the testing instructions to include mentioning that.
> 
> And, if you're testing what it looks like currently make sure that the backend patch is not applied and then you don't have the API sandboxed.
> 
> This change and the backend change have to go together.

Yes, I am aware. That's how I tested it (with the backend diff). I was only referring to the examples we both shared above and the updated screenshots in the description, which were showing the shorter string. They are probably on some different term variation, so my bad?

I left a couple of comments [here](https://github.com/Automattic/wp-calypso/pull/86434#discussion_r1789913879) and [here](https://github.com/Automattic/wp-calypso/pull/86434#discussion_r1789915153) though if you can take a look? I don't understand the motivation for changing these descriptions, especially with a "we should bring them back" note. 🤔

**Comment by sirbrillig** [comment]
@chriskmnds Thank you for the comments! I'm sorry. I may have misunderstood what you were looking for.

Basically, my motivation for this PR is to remove the current discount hack that is in place for these currencies and convert them to introductory offers; however, without this PR, introductory offers are not displayed correctly on the plans page. I'm not worried about the specifics of the text at the moment as long as it's accurate. The text can always be updated.

The reason I had changed the text in 2d2040d to remove the last line was so that the text for the new introductory offers would remain unchanged from how it looks currently. However, all the other strings returned by `usePlanBillingDescription()` for introductory offers display the regular recurring price, and by removing the regular price just for yearly plans, I figured someone would wonder why it was different from the others, which is why I had left the note comment to direct them to this discussion.

I've reverted 2d2040d so that all the strings returned by `usePlanBillingDescription()` are once again the same format. I'm happy to change the strings to whatever you'd like, though. Just let me know how you want it to read.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16898984

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @michaeldcain for including a screenshot in the description! This is really helpful for our translators.

**Comment by niranjan-uma-shankar** [comment]
FYI that the strings visible in production are not translated. 

The i18n bot says that `ja` is machine-translated but I don't see it. cc @Automattic/i18n 

Screenshot 2024-10-14 at 10 50 55 AM

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16898984).

**Comment by dlind1** [comment]
> FYI that the strings visible in production are not translated.
> 
> The i18n bot says that `ja` is machine-translated but I don't see it. cc @Automattic/i18n
> 

Looks good to me now. We've had an issue with importing new originals into GlotPress and also weren't deploying the translations most of the last week. The Japanese translations were successfully deployed about 14 hours ago.
Screenshot 2024-10-15 at 01 06 14

**Pull request merged**
